### ISSUE TITLE
weird results

### ISSUE THREAD

**Issue by phoenares** [bermanmaxim/LovaszSoftmax#29]
Hi，thanks for your work. I have added the tensorflow version of lovaszsoftmax to my task，I trained the model with 
Cross entropy loss first, then fine tuned it with Cross entropy loss + lovaszsoftmax loss（the weights is 1:1）, the mIoU improved about 2%.
But when tested on videos it seems like that the model without lovaszsoftmax loss performs better, especially on recall.
Do you have any idea about this, thank you.

my task is  2-class lane segmentation
